### DISCUSSION TITLE
Protractor type script migration

### DISCUSSION THREAD

**Discussion by Durgavivek** [webdriverio/codemod#47] [Migration Issue]
I Want the command for migrating type script protractor project.

Do i need to migrate folder by folder ,

**Comment by christian-bromann** [comment]
> I Want the command for migrating type script protractor project.

There are some known issues regarding TypeScript as it is not fully supported yet. Any contributions and support on this one are appreciated.

> Do i need to migrate folder by folder ,

Yes, you can run Protractor and WebdriverIO in parallel and migrate bit by bit.
